twittermap/public/stylesheets/main.css

```diff
@@ -18,7 +18,7 @@ body {
   bottom: 0;
   left: 96%;
   width: 24%;
-  height: 700px;
+  height: 90%;
```

Comment:
I changed this to suit alert-bar ver 1 (on the bottom) but for the new one it's unnecessary so I removed this change in new pull request.